### DISCUSSION TITLE
`ethereum.org` should be alway lowercase letter as a proper noun even in a beginning of sentence

### DISCUSSION THREAD

**Discussion by HiroyukiNaito** [ethereum/ethereum-org-website#12669] [General]
As I raised the PR, #12626.
There are still several 'Ethereum.org' at the beginning of sentences.
I think it should always be a lowercase letter in the contents as a proper noun.

It would prevent inconsistencies in the translation
Some people mistakenly regard it as a different word

**Comment by Bosemi** [comment]
> As I raised the PR, #12626. There are still several 'Ethereum.org' at the beginning of sentences. I think it should always be a lowercase letter in the contents as a proper noun.
> 
> It would prevent inconsistencies in the translation Some people mistakenly regard it as a different word

🚀 Exciting news for devs! The Auto Horizon Developer Challenge is live—build AI-powered decentralized apps using the Auto SDK and win up to $1,000 USDC!

🛠 Features: Decentralized storage, consensus tools, and more!
📅 Submit your project by Jan 31, 2025.

Details & Resources:

GitHub Repo: https://github.com/autonomys/auto-sdk
Join the Community: https://discord.gg/autonomys
Let’s innovate together and build the future of decentralized AI! 🌟
